get_github_coderabbit_review/tests/unit/test_resolved_marker.py

```
@@ -437,18 +437,18 @@ def test_process_threads_with_resolution(self):
 
     def test_update_config(self):
         """Test configuration update."""
-        original_marker = self.manager.config.default_marker
+        original_marker = self.manager.config.resolved_marker
```

Ruff (0.13.2)

Local variable `original_marker` is assigned to but never used

Remove assignment to unused variable `original_marker`

(F841)